aw_server/main.py

```diff
@@ -74,6 +76,11 @@ def parse_settings():
         dest="cors_origins",
         help="CORS origins to allow (as a comma separated list)",
     )
+    parser.add_argument(
+        "--custom-watcher-visualizations",
+        dest="custom_watcher_visualizations",
+        help="The custom watcher visualizations as a JSON string. The JSON contains a dict with key: watcher name and value: static folder path.",
+    )
```

Comment:
Maybe have the user pass a comma-separated list of key-value pairs (delimited by `=`) here instead, such as:

`--custom-static aw-watcher-custom=/home/user/code/aw-watcher-custom/static,...`

JSON is not really CLI-friendly, and should really be avoided in CLI arguments whenever possible.

Reply:
Okay,